solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpJdkSolrClient.java

```diff
@@ -165,8 +165,9 @@ public CompletableFuture<NamedList<Object>> requestAsync(
     }
   }
 
-  protected NamedList<Object> requestWithBaseUrl(
-      String baseUrl, SolrRequest<?> solrRequest, String collection)
+  @Override
+  public NamedList<Object> requestWithBaseUrl(
```

Comment:
change of argument order.  Was protected, now public implementing the abstraction.